mythril/laser/ethereum/state.py

```diff
@@ -56,28 +57,29 @@ def __init__(self, tx_id, starting_calldata=None):
             self.calldatasize = BitVec("{}_calldatasize".format(self.tx_id), 256)
             self.concrete = False
 
-        self.starting_calldata = starting_calldata or []
```

Comment:
I think this should be reflected in `native_contracts.py`

Reply:
All done, the modifications are reflected in `natives.py` now.